extensions/wikia/Email/templates/avatarLayout.mustache

```diff
@@ -1,6 +1,6 @@
 <div class="greeting">
 	<p>{{salutation}}</p>
-	<p>{{{articleEditedText}}}</p>
+	<p>{{{summary}}}</p>
```

Comment:
Gave this and the variable below a more generic name, since the current names are somewhat specific to the WatchedPage email.